ONSdigital/eq-questionnaire-runner
# Use placeholder default value

### What is the context of this PR?
Placeholder to use default value when answer is not provided

### How to review 

1. Launch survey `test_placeholder_default_value`
2. leave the textbox blank for no of employee question
3. Observe the next page shows the default value of 0
4. click continue and leave blank for employee training budget question
5. Observe the next page shows the default value of £250.00
6. Ensure all tests are passing

### Checklist
